examples/configuration/UsingSSHToConfigureNameResolution.ipynb

```diff
@@ -145,8 +213,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "SERVER_NAME='myServer'\n",
-    "\n",
     "echo Creating Server with fixed IP ${SERVER_IP}\n",
     "openstack server create --max-width 80 \\\n",
     "                        --flavor \"baremetal\" \\\n",
```

Comment:
If you add `--wait`, it will block until the server is ACTIVE.

Reply:
The idea is to spool up all those instances concurrently instead of in series. If you run `--wait` the jupyter notebook will wait for that step to complete, so starting another step will make jupyter said.

If you take a look at https://github.com/raubvogel/notebooks/blob/master/tutorials/getting-started/JupyterOrchestration.ipynb, I presented a way to keep an eye on the instances automagically until they reach an ACTIVE state. With that said, it is better for when you are automating the entire thing (which is why I wrote it to begin with).

Did I send a pull request for that file already?

Reply:
I don't think there's a pull request for that one yet